pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -395,6 +395,8 @@ type NodeGroup struct {
 	// +optional
 	VolumeType string `json:"volumeType"`
 	// +optional
+	RootDevice string `json:"rootDevice,omitempty"`
```

Comment:
I think we should probably consider something like `rootVolume.{name,type,size}` going forward, but for the time being, let's keep it consistent with existing fields, so perhaps `volumeName` would be most suitable.

Reply:
Cool, sounds good to me.